### DISCUSSION TITLE
Starting a discord community!!

### DISCUSSION THREAD

**Discussion by gaurangrshah** [supabase/supabase#927] [Feature Requests]
Not sure if this is on the roadmap at all, but I'd really love to see a discord community built around supabase. My personal reason for this is simple, I'm fairly new to the backend, and although supabase is a really great crutch -- I do frequently have questions that may really just have simple answers. And I although the discussions here on github are pretty good, I don't see the type of engagement that I get from other libraries and their communities. 

Just a thought, this by no means affects my usage of the library and am very happy with the experience. Thanks for all the great work you guys are doing! Looking forward to learning and growing with supabase as a weapon in my arsenal.

**Comment by silentworks** [comment]
One of the main issues with Discord is that its a closed platform. It's not easy to search for answers to questions and threading is still not great there. The benefit of discussions on Github is that its an open platform, searchable and also indexed by search engines. On here most questions are answered fairly quickly and its also easy to keep track of conversations.

**Reply by gaurangrshah** [reply]
Makes complete sense. Well I'll def. be asking more questions as they arise! Once again I really appreciate the work the supabase time is doing!

**Reply by kiwicopple** [reply]
> searchable and also indexed by search engines

This is 100% our motivation for using Discusssions. We were also considering Discourse before Discussions was released for this reason (but Discusssions is much easier for us to manage now)

We will open up a Discourse/Slack community at some point, but it's likely we will hire a community manager before then (to make sure that we can provide more instant responses!)
